middleware/validateProductData.js

```diff
@@ -0,0 +1,47 @@
+const Validator = require('validator')
+const isEmpty = require('is-empty')
+
+function validateProductInput (data) {
+  const errors = {}
+  // Convert empty fields to an empty string so we can use validator functions
+  let { name, description, price, stock } = data
```

Comment:
@DimejiAre You did not cater for images?

Reply:
@Rexben001 The plan was to have another model for images and link it using a productID foreign key as one product will have several images.